pkg/providers/pricing/zz_generated.pricing_aws.go

```diff
@@ -0,0 +1,375 @@
+//go:build !ignore_autogenerated
+
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pricing
+
+import "time"
+
+// generated at 2023-09-18T13:06:44Z for us-east-1
+
+var InitialPriceUpdateAWS, _ = time.Parse(time.RFC3339, "2023-09-18T13:06:44Z")
```

Comment:
```suggestion
var InitialPriceUpdateTimeAWS, _ = time.Parse(time.RFC3339, "2023-09-18T13:06:44Z")
```

Better to be specific that this is a time